Makefile

```diff
@@ -60,24 +60,25 @@ bootstrap: ## Prepare Docker images for the project
 bootstrap: \
 	data/media \
 	data/static \
-  env.d/development/crowdin \
+	env.d/development/crowdin \
+	env.d/terraform \
 	build \
 	run \
 	migrate \
 	i18n-compile
 .PHONY: bootstrap
 
 # -- Docker/compose
-build: ## build the app container
-	@$(COMPOSE) build app
+build: ## build the app-dev container
+	@$(COMPOSE) build app-dev
 .PHONY: build
 
 down: ## stop and remove containers, networks, images, and volumes
 	@$(COMPOSE) down
 .PHONY: down
 
-logs: ## display app logs (follow mode)
-	@$(COMPOSE) logs -f app
+logs: ## display app-dev logs (follow mode)
+	@$(COMPOSE) logs -f app-dev
 .PHONY: logs
 
 run: ## start the development server using Docker
```

Comment:
You should update `run` command to run only `postgresql` and `app-dev` services.

Reply:
Yes it could be interesting to run both production and development configuration.
In this case, I would inspired from `run` command of `richie-site-factory` because currently I think it's weird to run terraform services with this command.
https://github.com/openfun/richie-site-factory/blob/4926dcb6f5f2dac017439540cf8ea09bf4f20852/Makefile#L79-L85

Reply:
done :pray: